clients/venice-thin-client/src/test/java/com/linkedin/venice/client/schema/RouterBackedSchemaReaderTest.java

```diff
@@ -154,20 +154,18 @@ public void testGetValueSchemaWhenNotExists()
       Assert.assertEquals(schema1.toString(), VALUE_SCHEMA_1.toString());
       Mockito.verify(mockClient, Mockito.timeout(TIMEOUT).times(1)).getRaw(Mockito.anyString());
 
-      // If a missing schema is requested, we should not query it twice.
       Schema schema = schemaReader.getValueSchema(3);
       Assert.assertNull(schema);
-      Mockito.verify(mockClient, Mockito.timeout(TIMEOUT).times(2)).getRaw(Mockito.anyString());
-      Schema cachedSchema = schemaReader.getValueSchema(3);
-      Assert.assertNull(cachedSchema);
-      Mockito.verify(mockClient, Mockito.timeout(TIMEOUT).times(3)).getRaw(Mockito.anyString());
-
+      // Assert.assertThrows(VeniceClientException.class, () -> {
```

Comment:
Can you remove the commented out code?

Reply:
Actually can you alter the test so that initially it throw exception and then after a few times returns the expected answer?